apps/site/src/views/Home/Home.tsx

```diff
@@ -6,17 +6,21 @@ import Mentors from "./sections/Mentors/Mentors";
 import Sponsors from "./sections/Sponsors/Sponsors";
 import FAQ from "./sections/FAQ/FAQ";
 
+import StickerLayout from "./components/StickerLayout/StickerLayout";
+
 import styles from "./Home.module.scss";
 
 function Home() {
 	return (
-		<div className={styles.home}>
-			<Landing />
-			<About />
-			<Mentors />
-			<Sponsors />
-			<FAQ />
-		</div>
+		<StickerLayout>
```

Comment:
Consider making the sticker layer a child element of body. I think that API would be a bit more intuitive and less intrusive.